app/main/presenters/search_presenters.py

```diff
@@ -288,8 +288,18 @@ def _annotate_categories_with_selection(lot_slug, category_filters, request, url
             content_manifest, framework, aggregations, keys_to_remove, parent_category=category['value'])
 
         if selected_descendants:
-            # If a parentCategory has been sent as a url query param, de-select all other parent categories.
-            if request.values.get('parentCategory', category['value']) != category['value']:
+            # If a parentCategory has been sent as a url query param, and it's this category...
+            # (Note: `get` fallback is for if there are selected descendants but no parent category is set, which is
+            # not expected, but could happen if constructing URLs by hand.)
```

Comment:
Is the fallback scenario unlikely enough that we don't need a test for it? Or is already covered somewhere?

Reply:
The fallback was already in place: I was just documenting it, as it took me a while to understand why it was there. No, I don't think we test for it - it feels like an edge case, and seeing as it can't happen (I think!), it's probably okay not to test it.

The tests already run both these code paths, anyway, regardless of that fallback.